Homework lesson26 added.

iBubbleGun opened the pull request:
Homework lesson26 added.
Please, check it.

iBubbleGun committed:
Homework lesson26 added.

kartius commented on src/main/java/edu/hillel/homework/lesson26/repository/OrderRepository.java:
```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrderRepository {
```
It's a Service layer but not a repository

iBubbleGun replied:
Well done.

iBubbleGun committed:
Update homework lesson26.
Rename package name repository to services, class OrderRepository to OrderService and ProductRepository to ProductService.
